zio-http/src/main/scala/zhttp/service/Server.scala

```diff
@@ -31,7 +31,9 @@ sealed trait Server[-R, +E] { self =>
     case ConsolidateFlush     => s.copy(consolidateFlush = true)
   }
 
-  def make(implicit ev: E <:< Throwable): ZManaged[R with EventLoopGroup with ServerChannelFactory, Throwable, Unit] =
+  def make(implicit
+    ev: E <:< Throwable,
+  ): ZManaged[R with EventLoopGroup with ServerChannelFactory, Throwable, Start] =
```

Comment:
Needs unit tests :P

Reply:
@tusharmath I ran an scoverage report. We are already covering make using HttpRunnableSpec. I think what we should be doing instead is writing unit test cases for different server settings.

Reply:
Done.